server/src/og.ts

```diff
@@ -96,31 +99,32 @@ const getOGPageSubstack = async (subdomain: string, sld: string, landingPageSett
   }
 
   const { data } = await substackAxiosBase.get(`https://${url.host}/${path}`)
-  if (domainInfo.farcastEnabled) {
+
+  if (domainInfo.farcastMap) {
+    return await renderFarcasterMapFullTemplate(domainInfo)
+  } else if (!domainInfo.farcastBasicMap && !domainInfo.farcastText && !domainInfo.farcastPartial && !domainInfo.farcastSwap) {
+    return data
+  } else {
```

Comment:
can drop else

Reply:
Okay. it's my habit to put it in if else clause. Fixed now.